policies/deprecation-release-policy.md

```diff
@@ -0,0 +1,76 @@
+# Design System Library Deprecation and Release Policy
+
+## Overview
+
+This policy outlines release process to ensure smooth transitions and maintain the stability and reliability of the library. It also details the the procedures and timelines for deprecating components, utilities, and other features in the design system library.
+
+## Release Policy
+
+### Versioning
+
+We follow [Semantic Versioning](https://semver.org/) to manage releases:
+
+-   **Major Releases (X.0.0)**: Introduce breaking changes. Deprecated items are removed, and significant new features or changes are added.
+-   **Minor Releases (X.Y.0)**: Add new features and enhancements in a backward-compatible manner. May include deprecations.
+-   **Patch Releases (X.Y.Z)**: Include backward-compatible bug fixes and documentation updates.
+
+### Release Schedule
+
+-   **Breaking Change Releases**:
+    -   Breaking change releases will take place every 6 months.
+
+### Release Process
+
+1. **Pre-release**:
+
+    - **Feature Freeze**: Before a major or minor release, a feature freeze period is implemented to ensure stability.
+    - **Testing**: Comprehensive testing is conducted, including unit tests, integration tests, and user acceptance tests.
+
+2. **Release**:
+
+    - **Release Notes**: Detailed release notes are published with each release, documenting new features, bug fixes, deprecations, and breaking changes.
+    - **Versioning**: The new version is tagged in the repository and published, and the library is published to the package manager (e.g. npm).
```

Comment:
I was thinking this should probably be just this, npm isnt an example of a package manager we publish to, its the actual one
```suggestion
    - **Versioning**: The new version is tagged in the repository and published, and the library is published to the package manager npm.
```